platform/core/src/services/DisplaySetService/DisplaySetService.js

```diff
@@ -261,4 +266,45 @@ export default class DisplaySetService {
     }
     return allDisplaySets;
   }
+
+
+  filterDisplaySetInstances(displaySetInstanceUID, filteredInstances) {
```

Comment:
These are all public APIs, they should have documentation, AND also please add doc to the DisplaySetService on what is this filtering and how people can use it